src/Network-Protocols/SMTPClient.class.st

```diff
@@ -159,7 +159,8 @@ SMTPClient >> mailFrom: fromAddress [
 	" MAIL <SP> FROM:<reverse-path> <CRLF>"
 
 	| address |
-	address := (MailAddressParser addressesIn: fromAddress) first.
+	"MailAddressParser is not in the minimal image"
+	address := ((self environment at: #MailAddressParser) addressesIn: fromAddress) first.
```

Comment:
Same here. SMTP requires MailAddressParser. So if SMTP is in image then this parser should be here too.

Reply:
See #3999.